File: leetcode/Arrays_Hash/Missing Number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        count = [-1] * (len(nums) + 1)
        for i in range(len(nums)):
            count[nums[i]] = nums[i]
        for i in range(len(count)):
            if (count[i] == -1):
                return i
        return n

File: leetcode/Arrays_Hash/Missing Number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size();
        vector <int> count(n + 1, -1);
        for (int i = 0; i < n; i++){
            count[nums[i]] = nums[i];
        }
        for (int i = 0; i < count.size(); i++){
            if (count[i] == -1) return i;
        }
        return n;

    }
};